tests/functional/BackbeatConsumer.js

```diff
@@ -43,37 +39,43 @@ describe('BackbeatConsumer', () => {
                                           },
                                           bootstrap: true,
                                         });
-        zookeeper = zookeeperHelper.createClient(
-            zookeeperConf.connectionString);
-        zookeeper.connect();
-        zookeeper.on('ready', () => {
-            zookeeperReady = true;
-            _doneIfReady();
-        });
-        producer.on('ready', () => {
-            producerReady = true;
-            _doneIfReady();
-        });
-        consumer.on('ready', () => {
-            consumerReady = true;
-            _doneIfReady();
+        consumerConc = new BackbeatConsumer({
+            zookeeper: zookeeperConf,
+            kafka: kafkaConf, groupId: groupIdConc, topic,
+            queueProcessor,
+            concurrency: 10,
+            bootstrap: true,
         });
+        async.parallel([
+            done => producer.on('ready', done),
```

Comment:
I assume that having the outer function callback named `done` does not conflict with the `async` callbacks also named `done` - it was still confusing to read at a first glance xD.